breeze/breeze.d.ts

```diff
@@ -1,13 +1,15 @@
 // Type definitions for Breeze 1.4
 // Project: http://www.breezejs.com/
-// Definitions by: Boris Yankov <https://github.com/borisyankov/>, IdeaBlade <https://github.com/IdeaBlade/Breeze/>
```

Comment:
Maybe I shouldn't have removed that.  The contacts for individual changes are in the changelog below, though.